packages/react-liveness/src/components/FaceLivenessDetector/index.ts

```diff
@@ -10,4 +10,5 @@ export {
   AwsCredentialProvider,
   AwsTemporaryCredentials,
   AwsCredentials,
+  LivenessErrorState,
```

Comment:
What happens if we change this to a union type before exporting? Is that a breaking change?

Reply:
Think regardless of whether we move forward with the union the namespace should be reconsidered

Reply:
What namespace alternatives are you thinking of?

I think in terms of exporting it it wouldn't be breaking to my understanding since we don't export it at this time. What do you all think?

Reply:
This just `LivenessError` would suffice, and changing the keys from SCREAMING_SNAKE_CASE to camelCase, also maybe dropping the extraneous usage of ERROR in the keys, example:

```ts
enum LivenessErrorState = {
  MOBILE_LANDSCAPE_ERROR = 'MOBILE_LANDSCAPE_ERROR',
  MULTIPLE_FACES_ERROR = 'MULTIPLE_FACES_ERROR',
}

type LivenessError = 'mobileLandscape' | 'multipleFaces'
```